lib/dev/ui/spinner.rb

```diff
@@ -31,7 +31,7 @@ class Task
           def initialize(title, &block)
             @title = title
             @thread = Thread.new do
-              cap = Dev::UI::StdoutRouter::Capture.new(with_frame_inset: false, &block)
+              cap = Dev::UI::StdoutRouter::Capture.new(self, with_frame_inset: false, &block)
```

Comment:
It's passed to the thread block as `*@block_args`. It's because the `update_title` method needs to be accessible to the thread.

I wonder if it would make more sense to have it pass something like:

```
Capture.new(->(title) { update_title(title) }, ...
```

And then invoke like

```ruby
sg.add('foo') { |set_title| set_title.call('bar') }
```